numpy/fft/__init__.py

```diff
@@ -128,11 +128,13 @@
 Normalization
 -------------
 
-The default normalization has the direct transforms unscaled and the inverse
-transforms are scaled by :math:`1/n`. It is possible to obtain unitary
-transforms by setting the keyword argument ``norm`` to ``"ortho"`` (default is
-`None`) so that both direct and inverse transforms will be scaled by
-:math:`1/\\sqrt{n}`.
+The default normalization has the direct (forward) transforms unscaled and the
+inverse (backward) transforms scaled by :math:`1/n`. It is possible to obtain
+unitary transforms by setting the keyword argument ``norm`` to ``"ortho"``
+(default is ``"backward"``) so that both direct and inverse transforms are
+scaled by :math:`1/\\sqrt{n}`. Finally, setting the keyword argument "norm" to
+``"forward"`` has the direct transforms scaled by :math:`1/n` and the inverse
+transforms unscaled (i.e. directly oppposite to the default ``"backward"``).
```

Comment:
We should mention the old `None` here too:
```suggestion
transforms unscaled (i.e. exactly opposite to the default ``"backward"``).
`None` is an alias of the default option ``"backward"`` for backward compatibility.
```